### ISSUE TITLE
the normalize function in utils.py 

### ISSUE THREAD

**Issue by wangxiaoyunanne** [tkipf/pygcn#11]
Hi,
The normalize function in utils.py only normalize the row of adjacency matrix. While for the tensorflow version the implementation is different. You normalized both row and column. I am wondering will this lead to a difference for performance of GCN in accuracy ?

Best,
Xiaoyun

**Comment by tkipf** [comment]
Both versions result in similar performance but slight differences in
classification accuracy are possible.

**Comment by mathfinder** [comment]
Hi,
I want to know which version is better in the classification.

**Comment by tkipf** [comment]
Hard to say, it’s often best to try both variants.

**Comment by AllenWu18** [comment]
Well, I think in the function normalize(mx) it normalizes the features by row,
but if the node features are not one-hot form, and each node has its own feature 
like node1 : 1,3,7,20,57, which each feature represents different information( just like something similar in traditional machine learning, the different features has different meaning,  like age, height,  etc)
the each column of features has different dimension, 
should I normalize them by column, instead of row??

**Comment by tkipf** [comment]
Yes, you could normalize each feature to zero mean, unit standard deviation
per column.

**Comment by AllenWu18** [comment]
THX,I'll have a try
